playbooks/preflight-check.yml

```diff
@@ -138,3 +138,21 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+# RLM-1371 detect if enough space for rootfs backup
+- name: Check space for rootfs backup on infras
+  hosts: "os-infra_hosts"
+  user: root
+  tasks:
+    - name: Indentify the space used in /
+      shell: |
+        df -h / | tail -n 1 | awk '{print $5}' | sed 's/%//'
```

Comment:
Don't use root use /var/lib/lxc (or a respective override) in case the LXC stuff is on a sub volume